test/embedding/embedding_test.dart

```diff
@@ -175,5 +206,102 @@ main() {
         }
       },
     });
+    // Don't write the logs to file on a normal run.
+    expect(File(logFile).existsSync(), isFalse);
   });
 }
+
+String _filter(String input) {
+  return input
+      .replaceAll(p.toUri(d.sandbox).toString(), r'file://$SANDBOX')
+      .replaceAll(d.sandbox, r'$SANDBOX')
+      .replaceAll(Platform.pathSeparator, '/')
+      .replaceAll(Platform.operatingSystem, r'$OS')
+      .replaceAll(globalServer.port.toString(), r'$PORT')
+      .replaceAll(
+        RegExp(r'^Created:(.*)$', multiLine: true),
+        r'Created: $TIME',
```

Comment:
This is a lot of new hacks, just to a goldens test...
and that goldens test will break whenever we refactor code a bit, because it embeds a stacktrace...

Are we sure we want something this fragile?

Reply:
I was hoping that having a test of our stacktrace is a good thing. Hopefully it's easy to update the golden file.

I added a TODO, that we might reconsider this.

Reply:
Also I'm removing line-numbers from the stack-trace for making this a bit more stable over time :)